packages/js/src/ui/components/ExternalElementMounter.tsx

```diff
@@ -0,0 +1,26 @@
+import { onMount, ParentProps } from 'solid-js';
+
+type ExternalElementMounterProps = ParentProps<{
+  mount: (el: HTMLDivElement) => () => void;
+}>;
+
+export const ExternalElementMounter = ({ mount, ...rest }: ExternalElementMounterProps) => {
+  let ref: HTMLDivElement;
+
+  onMount(() => {
+    const unmount = mount(ref);
+
+    return () => {
+      unmount();
+    };
```

Comment:
You will need to use `onCleanup` method
This wont work in Solid :(

Reply:
indeed 😆